python_script/meter-health/meter_health.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import requests
+import csv
```

Comment:
Try to use Pep8 recommendations  in this section
[https://www.python.org/dev/peps/pep-0008/#imports](url)